addons/binding/org.openhab.binding.dlinksmarthome/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/

```

Comment:
Not sure what you mean by this, I had a quick look at other bindings and didn't see it included.